src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import { RelatedComponentsDecorator } from "../../../components";
+import { descriptionComponentsWithoutGeneralDescriptionMap } from "../../../components/related-components/component-description-map";
+import { Search } from "../../../../components";
+import { CatalogEmptyState } from "../EmptyState/Catalog.stories.EmptyState";
+import styles from "./Catalog.stories.module.scss";
+
+const RELATED_COMPONENT_NAMES = Array.from(descriptionComponentsWithoutGeneralDescriptionMap.keys())
```

Comment:
Nitpick: I don't think it should be constant

Reply:
Why not? Why to keep it in the component if it's not changing?

Reply:
It's depenedent on an external map

Reply:
Which is constant any way